cardano_node_tests/utils/configuration.py

```diff
@@ -35,6 +35,9 @@ def _check_cardano_node_socket_path() -> None:
 # See `cat /proc/sys/net/ipv4/ip_local_port_range`.
 PORTS_BASE = int(os.environ.get("PORTS_BASE") or 23000)
 
+SMASH_ADMIN = os.environ.get("SMASH_ADMIN", "admin")
+SMASH_PASSWORD = os.environ.get("SMASH_PASSWORD", "password")
+
```

Comment:
These are probably not needed here. The only place where these variables are used is in `smash_utils.py`, so you can read the env variables there.